steampy/client.py

```diff
@@ -181,12 +181,11 @@ def accept_trade_offer(self, trade_offer_id: str) -> dict:
             return self._confirm_transaction(trade_offer_id)
         return response
 
-    def _fetch_trade_partner_id(self, trade_offer_id: str) -> dict:
+    def _fetch_trade_partner_id(self, trade_offer_id: str) -> str:
         url = self._get_trade_offer_url(trade_offer_id)
         offer_response_text = self._session.get(url).text
         if "You have logged in from a new device. In order to protect the items" in offer_response_text:
```

Comment:
I use single quote in whole project

'You have logged in from a new device. In order to protect the items'